assets/chat/js/chat.js

```diff
@@ -1864,7 +1864,10 @@ class Chat {
   cmdEMBED(parts) {
     const { location } = window.top || window.parent || window;
     try {
-      location.hash = this.hashLinkConverter.convert(parts[0]);
+      location.hash = this.hashLinkConverter.convert(
+        parts[0],
+        /* throwErrors */ true
```

Comment:
Would you mind removing these comments? I think it's a little clumsy and doesn't offer very much utility.